src/components/ModalView.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {useState} from 'react';
+import {Alert, Modal} from 'react-native';
+import {
+  ModalViewCustom,
+  ModalContain,
+  ModalViewContain,
+  ModalTextError,
+  ModalText,
+  ButtonPressable,
+  ButtomStyle,
+} from '../styles/ModalViewStyles';
+
+const ModalView = () => {
+  const [modalVisible, setModalVisible] = useState(true);
+
+  const HandleViewModal = () => setModalVisible(!modalVisible);
```

Comment:
rename to handleViewModal 

and use the previous value of the state: 
 setModalVisible((prev) => !prev);